lib/checks/shared/focusable-not-tabbable.json

```diff
@@ -0,0 +1,11 @@
+{
+	"id": "focusable-not-tabbable",
+	"evaluate": "focusable-not-tabbable.js",
+	"metadata": {
+		"impact": "serious",
+		"messages": {
+			"pass": "No focusable elements contained within element",
+			"fail": "Focusable content should be removed or have tabindex='-1'"
```

Comment:
Change to `focusable content should have tabindex="-1" or be removed from the DOM` for consistency with the other check